Development/src/theme/ThemeContext.js

```diff
@@ -17,13 +17,18 @@ export const AppThemeProvider = ({ children }) => {
     });
     if (themeState.mode === undefined) setThemeState({ mode: 'light' });
 
-    const theme = createMuiTheme({
+    let theme = createMuiTheme({
```

Comment:
Theme declaration can be condensed to:
`const theme = responsiveFontSizes(createMuiTheme(`